libraries/Microsoft.Bot.Builder.Dialogs.Adaptive/Schemas/Actions/Microsoft.ForeachPage.schema

```diff
@@ -19,12 +19,12 @@
                 "$type": "Microsoft.IDialog"
             },
             "title": "Actions",
-            "description": "Actions to execute"
+            "description": "Actions to execute for each page."
```

Comment:
>page [](start = 56, length = 4)

Add $foreach.page and $foreach.item description

Reply:
Updated.